junit5/src/main/java/io/specto/hoverfly/junit5/HoverflyExtension.java

```diff
@@ -68,7 +69,7 @@ public void beforeAll(ExtensionContext context) {
         HoverflyConfig config = null;
 
         if (isAnnotated(annotatedElement, HoverflySimulate.class)) {
```

Comment:
In fact `isAnnotated` uses `findAnnotation` internally, so the branch is selected in case a meta-annotation is used these days. However, `annotatedElement.getAnnotation(...)` returns `null` in this case, leading to NPE.

So, this can be considered a bug fix =)

Reply:
kills two birds with one stone, nice fix 👍